herald/proxy.config.json

```diff
@@ -0,0 +1,8 @@
+{
+        "/herald/api/*": {
+                "target": "https://uthgard.org/",
+                "changeOrigin": true,
+                "logLevel": "debug",
+                "secure": false
```

Comment:
It handels the https certificate verification. Last time I read the nodejs code they included a bundle of ca-certificates within the binary. Depending on distribution/os this is running on it might still lack Lets Encrypt. Since that is only used during development it also eases switching the backend server to some local mock or whatever.